src/ucp/core/ucp_mm.c

```diff
@@ -1399,32 +1399,38 @@ static ucs_rcache_ops_t ucp_mem_rcache_ops = {
 
 static ucs_status_t
 ucp_mem_rcache_create(ucp_context_h context, const char *name,
-                      ucs_rcache_t **rcache_p, const ucs_rcache_config_t *config)
+                      ucs_rcache_t **rcache_p, ucs_rcache_params_t *rcache_params)
 {
-    ucs_rcache_params_t rcache_params;
-
-    ucs_rcache_set_params(config, &rcache_params);
-
-    rcache_params.region_struct_size = ucp_memh_size(context);
-    rcache_params.max_alignment      = ucs_get_page_size();
-    rcache_params.ucm_event_priority = 500; /* Default UCT pri - 1000 */
-    rcache_params.ucm_events         = UCM_EVENT_VM_UNMAPPED |
-                                       UCM_EVENT_MEM_TYPE_FREE;
-    rcache_params.context            = context;
-    rcache_params.ops                = &ucp_mem_rcache_ops;
-    rcache_params.flags              = UCS_RCACHE_FLAG_PURGE_ON_FORK;
-    rcache_params.alignment          = UCS_RCACHE_MIN_ALIGNMENT;
-
-    return ucs_rcache_create(&rcache_params, name, ucs_stats_get_root(),
+    rcache_params->region_struct_size = ucp_memh_size(context);
+    rcache_params->max_alignment      = ucs_get_page_size();
+    rcache_params->ucm_events         = UCM_EVENT_VM_UNMAPPED |
+                                        UCM_EVENT_MEM_TYPE_FREE;
+    rcache_params->context            = context;
+    rcache_params->ops                = &ucp_mem_rcache_ops;
+    rcache_params->alignment          = UCS_RCACHE_MIN_ALIGNMENT;
```

Comment:
why needed?

Reply:
@yosefe I had to add this back because gtests were failing. It seems that the UCP rcache instances do need alignment of UCS_RCACHE_MIN_ALIGNMENT compared to other ones as the alignment for those can/should be UCS_SYS_CACHE_LINE_SIZE.

Reply:
so the default should be UCS_RCACHE_MIN_ALIGNMENT, other places can override by UCS_SYS_CACHE_LINE_SIZE